frontend/dispatcher.lua

```diff
@@ -77,6 +77,7 @@ local settingsList = {
     calibre_search = { category="none", event="CalibreSearch", title=_("Search in calibre metadata"), device=true,},
     calibre_browse_tags = { category="none", event="CalibreBrowseTags", title=_("Browse all calibre tags"), device=true,},
     calibre_browse_series = { category="none", event="CalibreBrowseSeries", title=_("Browse all calibre series"), device=true, separator=true,},
+    edit_last_edited_file = { category = "none", event = "OpenLastEditedFile", title = 'Texteditor: open last file', device = true, separator = true, },
```

Comment:
```suggestion
    edit_last_edited_file = { category = "none", event = "OpenLastEditedFile", title =_("Texteditor: open last file"), device = true, separator = true, },
```

@smartscripts-nl please fix this in one of your other PRs

Reply:
I've done that, in #6608